core/templates/core/panel.html

```diff
@@ -0,0 +1,55 @@
+{% extends 'core/base.html' %}
+
+{% block title %}Panel Gwardii{% endblock %}
+
+{% load list_events %}
+
+{% block content %}
+    <div class="flex flex-row nowrap flex-grow h-full overflow-auto">
+        <div id="sidebar" class="hidden fixed top-0 md:relative md:block min-w-72 max-w-72 h-full bg-gray-800 shadow-2xl overflow-auto z-[0]">
```

Comment:
Wtf. Why to put `min-w-72` and `max-w-72`. Just use `w-72`

Reply:
No. for some reason it breaks.